docs/change-log/index.md

```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
+|ML-3520|MLRun does not decompress large Kubeflow pipelines                                                                                                                                   |NA                                                                                                                                                                                                                                                                                                                              |v1.3.0    |
+|ML-3824|MLRun supports TensorFlow up to 2.11.                                                                                                                                                |NA                                                                                                                                                                                                                                                                                                                              |v1.3.1    |
 |ML-3731|When trying to identify a failed step in a workflow with `mlrun.get_run_db().list_pipelines('project-name')`, the returned error is `None`.                                          |To see the error, use `mlrun.db.get_pipelines()` instead.                                                                                                                                                                                                                                                                        |
 |ML-3743|Setting AWS credentials as project secret cause a build failure on EKS configured with ECR.                                                                                          |When using an ECR as the external container registry, make sure that the project secrets AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY have read/write access to ECR, as described in the [platform documentation](https://www.iguazio.com/docs/latest-release/services/app-services/docker-registry/#create-off-cluster-registry)|
-|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.                                                                                                                  |
+|ML-4386|Local notifications aren’t persisted in mlrun API| NA | v1.5.0|
+|ML-4767|Use PyTorch versions up to and including than 2.0.1, but not higher.       | NA |                 v1.5.0|   
+|ML-4960|The Memberships tab in Projects > Settings is part of MLRun only when integrated with the platform.            | NA |                 v1.5.0| 
```

Comment:
The issue is with the caching, the fact that it's only part of the platform is not a limitation